modules/RTC/ortc/README.md

```diff
@@ -0,0 +1,69 @@
+* VP8 video produced by Edge is terribly rendered in Chrome/Firefox (it may be related to RTP mangling done in the JVB, no idea). See [screenshot](https://github.com/jitsi/lib-jitsi-meet/pull/473#issuecomment-300573271).
+* When another browser (Chrome/Firefox) in the conference reloads the web (it correctly dissapears and appears again in Edge) its previous audio/video tracks are correctly stopped and removed, the new audio plays well in Edge, but the new video produces an `InvalidState` error when creating a new `RTCRtpReceiver` for it and calling `rtpReceiver.receive(parameters)`. AFAIS the SSRC values of the new remote video do not match any previous one, but must investigate it further.
+
+
+## Files in `ortc` folder
+
+
+### RTCPeerConnection.js
+
+Exports a `RTCPeerConnection` shim.
+
+The interface is based on the [W3C specification of 2015](https://www.w3.org/TR/2015/WD-webrtc-20150210/), which matches (mostly) the current implementation of Chrome.
+
+It also implements Plan-B for multi-stream.
+
+
+#### Limitations
+
+* BUNDLE is assumed (single transport for all the local and remote media streams).
+* `rtcp-mux` is assumed (not a real problem nowadays).
+* Calling `createOffer()` is not (yet) implemented, so P2P mode is not supported (`RTCBrowserType.isP2PSupported()` returns `false`).
+* Calling `setRemoteDescription()` with a SDP answer with mangled SSRC values is currently unsupported (those new SSRC values will not be used for sending media).
+* If the app calls `createAnswer()`, mangles SSRC values, and applies by calling `setLocalDescription()`, those new SSRC values are currently ignored.
+* Simulcast not supported (currently Edge supports `maxFramerate` per encoding but it does not support `maxBitrate`, `resolutionScale` or `framerateScale`, so it's not worth it).
+* `RTCDataChannel` not supported (Edge does not implement it).
+* `getStats()` not yet implemented.
+* `addIceCandidate()` not supported (it's never called anyway, at least in non P2P mode).
+  - Edge does not support Trickle-ICE which means that, after reading all the remote ICE candidates from the remote SDP and applying them, an "empty" candidate must be immediately applied (otherwise the `RTCIceTransport` never enters the "completed" state) and, after that, new remote candidates cannot be added.
+
+
+### RTCSessionDescription.js
+
+Exports a `RTCSessionDescription` shim.
+
+The interface is the same as in the WebRTC specification (although internally it also handles a Object representation of the SDP generated by the [sdp-transform](https://www.npmjs.com/package/sdp-transform) library).
+
+
+#### Limitations
+
+* Cannot use `JSON.stringify()` with a `RTCSessionDescription` instance as argument. Instead, both `type` and `sdp` members must be directly retrieved (this is not a problem within the whole `lib-jitsi-meet` library). Also, a `toJSON()` method has been added (not standard, but implemented by Chrome and Firefox).
```

Comment:
I think it's worth noting that one cannot do `JSON.stringify(description)` and expect a pure JSON string having just `type` and `sdp`. If you don't care about that confirm it to me and I'l remove the limitation from the README.